analyzer/src/funTest/assets/projects/synthetic/conda-django-expected-output.yml

```diff
@@ -0,0 +1,146 @@
+---
+project:
+  id: "Conda:::<REPLACE_REVISION>"
+  definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/python3-django/environment.yml"
+  declared_licenses: []
+  declared_licenses_processed: {}
+  vcs:
+    type: ""
+    url: ""
+    revision: ""
+    path: ""
+  vcs_processed:
+    type: "Git"
+    url: "https://github.com/heremaps/oss-review-toolkit.git"
+    revision: "<REPLACE_REVISION>"
+    path: "analyzer/src/funTest/assets/projects/synthetic/python3-django"
```

Comment:
Also use "<REPLACE_URL>" and "<REPLACE_PATH>" above. This ensures that the tests also succeeds if it is cloned from another location, e.g. a fork.